### ISSUE TITLE
Add esriRelationship field

### ISSUE THREAD

**Issue by hhkaos** [esri-es/arcgis-experts#14]
I think it could make sense to add this field with this information and these possible values: employee/distributor/partner/customer/community. 

Don't you think?

**Comment by mschenker** [comment]
There can be reasons to do this, but then we should first have a discussions about the goals of this list.
I do not think that an "esriRelationship" field contributes to the goals stated in [https://github.com/esri-es/arcgis-experts#why-this-project
](https://github.com/esri-es/arcgis-experts#why-this-project
)

**Comment by hhkaos** [comment]
You are right @mschenker !, I was thinking on it to "reward" or increase the visibility of people not working at Esri or distributors. But like [the location](https://github.com/esri-es/arcgis-experts/issues/10), I think I would add it as an advance parameter filter because I would like to keep the interface as clean as simple as possible (and of course at it to the profile showing in the modal).

Does it make sense? If it does I can update the list of goals adding this "reward" thing ;).

This issue gave me [another idea](https://github.com/esri-es/arcgis-experts/issues/17)
